src/Config.ts

```diff
@@ -38,22 +40,6 @@ export interface Config {
  */
 export const Config = Context.Tag<Config>()
 
-const ConfigSchema = Schema.struct({
```

Comment:
I would export these schemas instead of adding another module

Reply:
I tried that actually.. but trying to import ConfigSchema from the gen-schema script causes an error. 

```
> @effect/docgen@0.2.0 gen-schema /Users/jessekelly/projects/clones/docgen
> ts-node scripts/gen-schema.ts

/Users/jessekelly/projects/clones/docgen/node_modules/.pnpm/ts-node@10.9.1_@types+node@20.4.8_typescript@5.1.6/node_modules/ts-node/dist/index.js:851
            return old(m, filename);
                   ^
Error [ERR_REQUIRE_ESM]: require() of ES Module /Users/jessekelly/projects/clones/docgen/node_modules/.pnpm/chalk@5.3.0/node_modules/chalk/source/index.js from /Users/jessekelly/projects/clones/docgen/src/Config.ts not supported.
Instead change the require of index.js in /Users/jessekelly/projects/clones/docgen/src/Config.ts to a dynamic import() which is available in all CommonJS modules.
    at require.extensions.<computed> [as .js] (/Users/jessekelly/projects/clones/docgen/node_modules/.pnpm/ts-node@10.9.1_@types+node@20.4.8_typescript@5.1.6/node_modules/ts-node/dist/index.js:851:20)
    at Object.<anonymous> (/Users/jessekelly/projects/clones/docgen/src/Config.ts:35:33)
    at m._compile (/Users/jessekelly/projects/clones/docgen/node_modules/.pnpm/ts-node@10.9.1_@types+node@20.4.8_typescript@5.1.6/node_modules/ts-node/dist/index.js:857:29)
    at require.extensions.<computed> [as .ts] (/Users/jessekelly/projects/clones/docgen/node_modules/.pnpm/ts-node@10.9.1_@types+node@20.4.8_typescript@5.1.6/node_modules/ts-node/dist/index.js:859:16)
    at Object.<anonymous> (/Users/jessekelly/projects/clones/docgen/scripts/gen-schema.ts:28:18)
    at m._compile (/Users/jessekelly/projects/clones/docgen/node_modules/.pnpm/ts-node@10.9.1_@types+node@20.4.8_typescript@5.1.6/node_modules/ts-node/dist/index.js:857:29)
    at require.extensions.<computed> [as .ts] (/Users/jessekelly/projects/clones/docgen/node_modules/.pnpm/ts-node@10.9.1_@types+node@20.4.8_typescript@5.1.6/node_modules/ts-node/dist/index.js:859:16)
    at phase4 (/Users/jessekelly/projects/clones/docgen/node_modules/.pnpm/ts-node@10.9.1_@types+node@20.4.8_typescript@5.1.6/node_modules/ts-node/dist/bin.js:466:20)
    at bootstrap (/Users/jessekelly/projects/clones/docgen/node_modules/.pnpm/ts-node@10.9.1_@types+node@20.4.8_typescript@5.1.6/node_modules/ts-node/dist/bin.js:54:12)
    at main (/Users/jessekelly/projects/clones/docgen/node_modules/.pnpm/ts-node@10.9.1_@types+node@20.4.8_typescript@5.1.6/node_modules/ts-node/dist/bin.js:33:12)
    at Object.<anonymous> (/Users/jessekelly/projects/clones/docgen/node_modules/.pnpm/ts-node@10.9.1_@types+node@20.4.8_typescript@5.1.6/node_modules/ts-node/dist/bin.js:579:5) {
  code: 'ERR_REQUIRE_ESM'
}
 ELIFECYCLE  Command failed with exit code 1.
```

Reply:
It seems to be caused by chalk.

Reply:
yeah, chalk 5 is esm only, you can use `tsx` instead of `ts-node` to run `gen-schema`

```sh
pnpm add tsx -D
```

```json
{
  "gen-schema": "tsx scripts/gen-schema.ts"
}
```

Reply:
Should we remove node-ts and just use it for both scripts(assuming it works, I haven't tested it yet)?

Reply:
yes we can use tsx for both scripts (we can't remove ts-node though as it's used as dependency)

Reply:
Cool! Updated!